index.js

```diff
@@ -251,15 +240,23 @@ function BigAssFanAccessory(log, config) {
     return (value ? Characteristic.RotationDirection.COUNTER_CLOCKWISE : Characteristic.RotationDirection.CLOCKWISE);
   }
 
+  var occupancySetWrapper = function(value) {
+    return value == Characteristic.OccupancyDetected.OCCUPANCY_NOT_DETECTED;
+  }
+  
   var occupancyGetWrapper = function(value) {
     return (value ? Characteristic.OccupancyDetected.OCCUPANCY_DETECTED : Characteristic.OccupancyDetected.OCCUPANCY_NOT_DETECTED);
   }
 
   var lightMaxBrightness = this.myBigAss.light.max ? this.myBigAss.light.max : 16;
   var fanMaxSpeed        = this.myBigAss.fan.max ? this.myBigAss.fan.max : 7;
-
-  this.lightService = new Service.Lightbulb(this.homekitLightName);
-
+  
+  var existingLightBulbService;
+  if (existingAccessory){
+	  existingLightBulbService = existingAccessory.getService(this.homekitLightName);
```

Comment:
If you wouldn't mind making these changes use spaces instead of tabs for the indentation I'd really appreciate it 👍